membership_account_balance/README.rst

```diff
@@ -0,0 +1,27 @@
+Membership Management - Member Account Balance
```

Comment:
@elicoidal  Do you know which project_repo does these module push to? I shoud add the project_repo in the README.rst.